developers/prompt-scavenger.md

```diff
@@ -1,8 +1,8 @@
 ---
 description: Learn how to interact with the Celestia Node API with this tutorial.
 prev:
-  text: "Node RPC CLI tutorial"
-  link: "/developers/node-tutorial"
+  text: "Golang client tutorial"
+  link: "/developers/golang-client-tutorial"
```

Comment:
the `prev` and `next` frontmatter allow you to customize the text/link for previous and next pages that are shown at the footer. this is something i mostly use here to skip external links in the menu, preserving the nav flow ([page on it in vitepress docs](https://vitepress.dev/reference/default-theme-prev-next-links#prev-next-links))